script.js

```diff
@@ -1,12 +1,44 @@
-let colorlist = ['gold', 'yellow', 'turquoise', 'red']
+let img;
+let backgroundImg;
+function preload() {
+  img = loadImage('./images/turtle.svg');
+  backgroundImg = loadImage('./images/beach.svg')
+}
 
 function setup() {
-  createCanvas(windowWidth, windowHeight);
-    background(255);
+  createCanvas(displayWidth, displayHeight);
+  backgroundImg.resize(windowHeight,windowWidth - 200)
```

Comment:
I think you have these the wrong way round, which is why the background is slightly bigger than the window. Should be `backgroundImg.resize(windowWidth, windowHeight)`